src/components/pages/devices/flyouts/deviceJobs/deviceJobTags.js

```diff
@@ -34,7 +34,7 @@ import {
 
 update.extend('$autoArray', (val, obj) => update(obj || [], val));
 
-const isNumeric = value => typeof value === 'number' || !isNaN(parseInt(value, 10));
+const isNumeric = value => typeof value === 'number';
```

Comment:
Is it guaranteed that isNumeric will only be used on values that are actually numbers? If a string value comes in that represents a number var num = "30", do we want that to also be a number? Which I think is why the !isNaN was there. Just want to make sure it isn't being used in this way.

Reply:
Just because a value looks like a number doesn't mean that the actual type is a number. The tags and properties can be either text or number... and there is a drop down for the user to specify which one they want.

Reply:
BTW when use selects a number but enters a string do we check?